index.html

```diff
@@ -3885,6 +3885,29 @@ <h3>JSON-LD</h3>
 base media type, `application/vc+ld+json`.
         </p>
 
+        <p>
+As elaborated upon in Section <a href="#static-processing"></a>, some software
+applications might not perform dynamic JSON-LD processing, which utilizes
+features such as JSON-LD compaction and expansion. Static processors will
+elect to, for example, reject unrecognized URL values in the
+`@context` property. <a>Conforming document</a> authors are also advised that
+usage of certain dynamic JSON-LD processor features might reduce
+interoperability. These features include:
+        </p>
+
+        <ul>
+          <li>
+Use of JSON-LD keywords in the `@context` value that globally modify document
+value processing, such as global settings of `@base` and `@language`
+          </li>
+          <li>
+Using full URLs for JSON-LD terms and types instead of the compact form of those
+values defined in application-specific JSON-LD context files
```

Comment:
Replace with
"Use of full URLs for JSON-LD terms and types, instead of the compact forms of
those values which may or may not be defined in application-specific JSON-LD context files"

Reply:
This suggestion makes no sense in context of the existing thread (started by [your 19 hour older comment](https://github.com/w3c/vc-data-model/pull/1276#discussion_r1375782617), followed on by [my 10 hour older comment](https://github.com/w3c/vc-data-model/pull/1276#discussion_r1376988222)). The verb tense here ("Using") breaks from the intro preceding the `<ul>` and differs from that in both bullets including this one. I had much to say about your "values which may or may not be defined"/"values whether or not they are defined" in [my previous comment](https://github.com/w3c/vc-data-model/pull/1276#discussion_r1376988222), but that comment appears to have been ignored completely.

Reply:
I get your point. But I am not sure you get the point I am trying to make, which is: an issuer that only uses the default at context files, but defines their own application types, wishes to use the full URLs for these, since the VCDM requires them to be URLs. The issuer does not want to use a local string that maps via the at vocab into a URL, since another issuer could use the same local string that maps into the same URL via the at vocab, whilst it has a different semantic. Using the full URL is conformant to the VCDM but the proposed text says that it might affect interoperability. However using a local string could cause a bigger car crash because recipients would interpret two different application types to be the same one by mapping them both via the at vocab.

Reply:
I believe I understand what you're trying to say here. I think it may not be possible (nor necessary) to wedge all that nuance into this list item. I have a [new draft for the list item](https://github.com/w3c/vc-data-model/pull/1276/files#r1382158148), which changes both existing lines, so it's not going directly on this thread (which is only attached to one line of text). I suggest trying to craft a new list item, if the existing list items do not now cover your concern as elaborated above.

Reply:
Alternatively we could change the text of this PR to be more precise, which you could help by wordsmithing the following e.g.

"using full URLs for JSON-LD terms and types instead of the compact form of those
values that are either explicitly defined in application-specific JSON-LD context files, or are implicitly defined by the  `@vocab` feature of the base `@context` file `https://www.w3.org/ns/credentials/v2`